Tests/XCUITests/ThirdPartySearchTest.swift

```diff
@@ -120,30 +128,34 @@ class ThirdPartySearchTest: BaseTestCase {
         app.navigationBars["Settings"].buttons["AppSettingsTableViewController.navigationItem.leftBarButtonItem"].tap()
     }
 
-    func testCustomEngineFromIncorrectTemplate() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        waitForTabsButton()
-        navigator.nowAt(NewTabScreen)
-        navigator.goto(AddCustomSearchSettings)
-        app.textViews["customEngineTitle"].tap()
-        app.typeText("Feeling Lucky")
-
-        UIPasteboard.general.string = "http://www.google.com/search?q=&btnI"
-
-        let tablesQuery = app.tables
-        let customengineurlTextView = tablesQuery.textViews["customEngineUrl"].staticTexts["URL (Replace Query with %s)"]
-
-        XCTAssertTrue(customengineurlTextView.exists)
-        customengineurlTextView.tap()
-        customengineurlTextView.press(forDuration: 2.0)
-        app.staticTexts["Paste"].tap()
-
-        waitForExistence(app.buttons["customEngineSaveButton"], timeout: 3)
-        app.buttons["customEngineSaveButton"].tap()
-        waitForExistence(app.navigationBars["Add Search Engine"], timeout: 3)
-        app.navigationBars["Add Search Engine"].buttons["Save"].tap()
-
-        waitForExistence(app.alerts.element(boundBy: 0))
-        XCTAssert(app.alerts.element(boundBy: 0).label == "Failed")
+    func testCustomEngineFromIncorrectTemplate() throws {
+        if processIsTranslatedStr() == m1Rosetta {
+            throw XCTSkip("Copy & paste may not work on M1")
+        } else {
+            navigator.performAction(Action.CloseURLBarOpen)
+            waitForTabsButton()
+            navigator.nowAt(NewTabScreen)
+            navigator.goto(AddCustomSearchSettings)
+            app.textViews["customEngineTitle"].tap()
+            app.typeText("Feeling Lucky")
+            
+            UIPasteboard.general.string = "http://www.google.com/search?q=&btnI"
+            
+            let tablesQuery = app.tables
+            let customengineurlTextView = tablesQuery.textViews["customEngineUrl"].staticTexts["URL (Replace Query with %s)"]
+            
+            XCTAssertTrue(customengineurlTextView.exists)
+            customengineurlTextView.tap()
+            customengineurlTextView.press(forDuration: 2.0)
+            app.staticTexts["Paste"].tap()
```

Comment:
I have not been successful in typing in `customengineurlTextView` like the change in `FxScreenGraph`. The `textView` cannot be focused even after I `tap()` and `sleep()` before checking `hasFocus` or attempting to `typeText`. 😞

Reply:
That name is a bit confusing, the `customengineurlTextView` field is actually a `statictText` view...
`customengineurlTextView = tablesQuery.textViews["customEngineUrl"].staticTexts["URL (Replace Query with %s)"]`
if you want to type into the text field you have to use: 
`tablesQuery.textViews["customEngineUrl"].typeText("foo bar...")`
I have tried that locally and it works, I would give this a try before disabling the full test case.

Reply:
You are right! `typeText` works on `textViews` and not on `staticTexts`. 🚀

Reply:
I have re-enabled the test. 🎉